index.bs

```diff
@@ -1252,7 +1285,7 @@ A user agent MUST dispatch a <dfn event for="XRSession">selectend</dfn> event on
 
 A user agent MUST dispatch a <dfn event for="XRSession">select</dfn> event on an {{XRSession}} when one of its {{XRInputSource}}s has fully completed a [=primary action=]. The event MUST be of type {{XRInputSourceEvent}}.
 
-A user agent MUST dispatch a <dfn event for="XRFrameOfReference">boundschange</dfn> event on an {{XRFrameOfReference}} when the [=stage bounds=] change. This includes changes to the {{geometry}} point array or the {{bounds}} attribute changing to or from <code>null</code>. The event MUST be of type {{XRCoordinateSystemEvent}}.
+A user agent MUST dispatch a <dfn event for="XRReferenceSpace">reset</dfn> event on an {{XRReferenceSpace}} when the position or orientation of the origin changes relative to the user's environment. It also fires when the {{boundsGeometry}} changes for a {{XRBoundedReferenceSpace}}. The event MUST be of type {{XRReferenceSpaceEvent}}, and MUST be dispatched prior to the execution of any [=XR animation frame=]s that make use of the new origin.
```

Comment:
Erk.  That's a bit too far in the other direction.  The way this is written, the event would fire practically every frame on a device with inside-out tracking ^_^

Reply:
🤔 I get what you're saying, but I'm not sure I have better phrasing in mind? (Aside from maybe saying "*large* changes"?) Any suggestions?

Reply:
Alright, gave it another try:

> MUST dispatch a `reset` event on an `XRReferenceSpace` when discontinuities of the origin occur. (That is, significant changes in the origin's position or orientation relative to the user's environment.)

If you still feel it's not accurate I'd really appreciate some suggested verbiage.

Reply:
I suspect that's as good as we're going to get